packages/grafana-data/src/text/markdown.ts

```diff
@@ -42,3 +42,25 @@ export function renderTextPanelMarkdown(str?: string, options?: RenderMarkdownOp
 
   return sanitizeTextPanelContent(html);
 }
+
+// Markdown for Grafana Chats only!
+export function renderChatMarkdown(str?: string, options?: RenderMarkdownOptions): string {
```

Comment:
As a simple first step... move this function into:

packages/grafana-runtime/src/components/Chat.tsx

Reply:
Done in 52906830fb1ae0dff4049d7053c255f3aa59a6cd

So I placed func near Chat component, this required adding `marked` to Grafana package.json. Also, using second argument in `marked` function to set custom options per-call and avoid setting options on global marked instance. Hope this is fine.

Reply:
CI now fails – and I don't understand why :( https://github.com/grafana/grafana/runs/5107487642?check_suite_focus=true